infrastructure/bin/post_deployment.sh

```diff
@@ -71,7 +68,8 @@ fi
 add_section_block "$TRIPLE_BACK_TICK $CLEANED_STDOUT $TRIPLE_BACK_TICK"
 
 # Load Laravel Scheduler cron
-if echo "  *  *  *  *  * root    cd /home/site/wwwroot/api && php artisan schedule:run" >> /etc/crontab ; then
+# For extra debugging you can add `>> /tmp/run_laravel_scheduler.log 2>&1` to the end of the cron'd command
+if echo "  *  *  *  *  * root    . /etc/profile ; php /home/site/wwwroot/api/artisan schedule:run" >> /etc/crontab ; then
```

Comment:
Oh man, this took me so long to figure out.  Turns out that processes launched from cron do not automatically have all the environment variables and they have to be added in manually.